data/en/bots/null.md

```diff
@@ -0,0 +1,20 @@
+---
+avatar: 'https://i.imgur.com/SulQ5mn.png'
+cover: null
+custom_path: null
+description: >-
+  Secure your Discord Server - the strongest ban enforcement system, protection
+  against scraping and bots - without privacy issues.
+github: null
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=496677692284993546&scope=bot&permissions=355571&redirect_uri=https%3A%2F%2Fcheckpoint.discord.id%2Fadded%2F&response_type=code
+nsfw: false
+pagename: Discord Checkpoint
+prefix: cp!
+support: 'https://discord.gg/bGcjZRz'
```

Comment:
You may wish to create another server just for support - It just opens the checkpoint with no channels to talk in

Reply:
Hm, is this a actual problem?

Reply:
Well the button is for technical support with the bot

If users can't find technical support, it's not useful as a button